include/certifier.h

```diff
@@ -50,23 +51,29 @@ const key_message* GetPublicPolicyKey();
 
 bool PublicKeyFromCert(const string& cert, key_message* k);
 
+bool GetParentEvidence(const string& enclave_type,
+                       const string& parent_enclave_type,
+                       string* out);
```

Comment:
Refactored long single-line declarations that were going beyond 80-chars or were line-wrapped with one/two tab-stops to be in this multi-line prototype declarations format.